README.md

```diff
@@ -1,9 +1,99 @@
 
-# MewsFeed
+# MewsFeed – for the calm cats
```

Comment:
There's several places where we're using the old capitalization "MewsFeed" instead of "mewsfeed"